lib/alephant/storage.rb

```diff
@@ -95,9 +95,18 @@ def add_custom_meta(object)
       }
     end
 
+    def override_host_path?
+      ENV['AWS_S3_HOST_OVERRIDE'] == 'true'
+    end
+
+
     def client
       options = {}
       options[:endpoint] = ENV['AWS_S3_ENDPOINT'] if ENV['AWS_S3_ENDPOINT']
+      if override_host_path?
+        options[:disable_host_prefix_injection] = true
```

Comment:
Probably want a test around this just checking the right options are set for each path.

Reply:
tested manually but good point

Reply:
done